### DISCUSSION TITLE
troll comments is being left in discussion

### DISCUSSION THREAD

**Discussion by rebbieboi** [owen-it/laravel-auditing#898] [General]
![image](https://github.com/owen-it/laravel-auditing/assets/714513/4b820ca3-c04b-4050-b1f1-716aac2c31d0)

This person is a troll who doesn't help anyone in the community. Can someone remove his comment from a valid discussion with a correct answer?

**Comment by PaolaRuby** [comment]
just grow up, this sends unnecessary notifications to everyone, it's basically spam.
